Assets/Scripts/Plate/Plate.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Plate : MonoBehaviour
+{
+    [SerializeField] private AudioSource breakSound;
+    private bool _breakSoundCooldown = true;
```

Comment:
the name `breakDownCooldown` seems not like a boolean, so it should be `isBreakingDown` and the flag should be inverted with this naming as well